test/new-e2e/tests/cws/ec2_test.go

```diff
@@ -73,7 +73,7 @@ func TestAgentSuite(t *testing.T) {
 	)
 }
 
-func (a *agentSuite) TestOpenSignal() {
+func (a *agentSuite) Test00OpenSignal() {
```

Comment:
```suggestion
func (a *agentSuite) Test007OpenSignal() {
```
let's go full James Bond on this :trollface: